openpmd_validator/check_h5.py

```diff
@@ -702,20 +715,39 @@ def check_particles(f, iteration, v, extensionStates) :
     result_array = np.array([ 0, 0]) 
 
     # Find the path to the data
-    base_path = ("/data/%s/" % iteration).encode('ascii')
+    base_path = "/data/%s/" % iteration
```

Comment:
Why is the previously-defined `base_path` (used for the meshes) not kept here?

Reply:
it was inconsistent between `meshesPath` and `particlesPath`.
I unified both by just picking one.